lib/dotinstall/util.py

```diff
@@ -115,6 +115,31 @@ def create_symlink(source: str, target: str) -> None:
     os.symlink(source, target)
 
 
+def concat_files(sources, target, comment_prefix=None):
+    for i, _ in enumerate(sources):
+        sources[i] = os.path.abspath(os.path.expanduser(sources[i]))
+    target = os.path.expanduser(target)
+    _remove_symlink(target)
+    create_dir(os.path.dirname(target))
+    print('Concatenating %s to %s...' % (sources, target))
+    with open(target, 'w') as ft:
```

Comment:
Please pass the file through [black](https://github.com/psf/black)

```
python3 -m pip install --user black
black -l79 path/to/your/fork
```

Reply:
Let's name `ft` a `handle`

Reply:
Done.